db/migrate/20200425033724_create_existing_items.rb

```diff
@@ -0,0 +1,8 @@
+class CreateExistingItems < ActiveRecord::Migration[6.0]
+  def change
+    create_table :existing_items do |t|
+      t.string :idempotent_key
+      t.integer :item_type
```

Comment:
I'd just call this type instead of item_type since we already have an attribute called item_type and I wouldn't wanna confused it